pipelines/scatac-seq/run-local.sh

```diff
@@ -0,0 +1,4 @@
+#!/bin/bash
```

Comment:
The following .sh files are useful for developing locally but IMO shouldn't be committed to the repo.